formulario/admin.py

```diff
@@ -19,5 +19,16 @@ def get_queryset(self, request):
         qs = super(MapeamentoAdmin, self).get_queryset(request)
         return qs.filter(validacao=False)
 
-admin.site.register(Mapeamento, MapeamentoAdmin)
+class MapeamentoValidado(Mapeamento):
+    class Meta:
+        proxy = True
+
+class MapeamentoValidadoAdmin(MapeamentoAdmin):
+    def get_queryset(self, request):
+        qs = super(MapeamentoAdmin, self).get_queryset(request)
+        return qs.all()
```

Comment:
Aqui não deveríamos filtrar pelos mapeamentos que possuem `validacao=True`? Sem o filtro todos os mapeamentos, validados ou não, aparecem nessa listagem. Coloquei abaixo um GIF demonstrando :smile: 
```suggestion
        return qs.filter(validacao=True)
```

![Peek 2020-10-24 17-44](https://user-images.githubusercontent.com/14933043/97093297-09dad400-1621-11eb-91b6-6358c1b905ab.gif)

Reply:
De fato! Farei a correção.